app/scripts/controllers/reviewRegistration.js

```diff
@@ -164,10 +164,10 @@ angular
           return registration.completeRegistration(currentRegistration);
         })
         .then(function() {
-          //send analytics event
-          analytics.digitalData.eventID = conference.id;
-          analytics.digitalData.registeredEventName = conference.name;
-          analytics.track('registration');
+          analytics.track('registration', {
```

Comment:
Although I think this is a better way to do it, I think Clark just wants us to send the event name ```registration``` and then access the digital Data layer from the GTM side like we were doing or Adobe.

Reply:
Would setting both ways hurt? Seems like it'd be better not to mutate global state.

Reply:
Technically you can send the data and they can just ignore it on the gtm side, but still access the digitalData layer like before.